docs/algorithms.rst

```diff
@@ -112,28 +112,21 @@ Explicit solvent
 Solvent model
 ^^^^^^^^^^^^^
 
-.. warning:: Only TIP3P is supported through the :ref:`YAML method of setting up solvents <yaml_head>` at this moment. We are working on adding back in support for other water models before the YANK 1.0 release, but after the preview build
+Any explicit solvent model that can be constructed via AmberTools or that is distributed along with OpenMM is supported.
```

Comment:
Do we really support addition of solvent from OpenMM at this time? I thought it was only those solvents present in AmberTools.